Session2.Rmd

```diff
@@ -365,9 +379,10 @@ line_width <- c(0.25, 0.75, 0.5, 1)
 
 ```{r}
 ggplot(data = lines_Delft_selection) +
-  geom_sf(aes(size = highway)) +
-  scale_size_manual(values = line_width) +
-  ggtitle("Mobility network of Delft", subtitle = "Roads & Cycleways - Line width varies") + 
+  geom_sf(aes(linewidth = highway)) +
+  scale_linewidth_manual(values = line_width) +
```

Comment:
Please note the `scale_linewidth_manual()` is available since `ggplot 3.4.1`. Learners with older version might have problems with this.

Reply:
Or we just ask them to have the latest versions of everything installed.